### PR TITLE
Apply BEM refactoring for details

### PR DESCRIPTION
**Pull request by AlexanderSchmidtCE**
Refs: #7036

The A11y and PO reviews will only take place after all other DoD steps have been completed by the Developer:
- [x] Meaningful pull request title for the release notes
- [x] Pull request is linked to an issue and all changes relate to the issue
- [x] Tests to protect this code implemented (if applicable)
- [x] Manual test performed successfully (if applicable)
- [x] Documentation or migration has been updated (if applicable)

### CODE REVIEW

File: packages/components/src/components/details/style.scss
Comment (sdvg) [inline packages/components/src/components/details/style.scss:1]: In Zeile 25: `open` sollte ein BEM-Modifier sein

**Review by sdvg — CHANGES_REQUESTED** [review]

File: packages/themes/ecl/src/ecl-ec/components/accordion.scss [hunk 1]
```diff
@@ -13,15 +13,15 @@
...
+			.kol-button {
 				text-align: left;
 				width: 100%;
 				padding: 0 var(--spacing-m);
 				display: flex;
 				place-items: center;
 				line-height: 1.2;
 
-				span {
+				.kol-button-text {
```
Comment (sdvg) [inline packages/themes/ecl/src/ecl-ec/components/accordion.scss]: Gibt es diese Klasse? Sollte es nicht `kol-button__text` sein?

File: packages/themes/itzbund/src/components/accordion.scss [hunk 2]
```diff
@@ -20,7 +20,7 @@
 		&__heading-button {
 			@include kol-typography-accordion;
 
-			& button {
+			& .kol-button {
```
Comment (sdvg) [inline packages/themes/itzbund/src/components/accordion.scss]:
```suggestion
			.kol-button {
```

**Review by sdvg — CHANGES_REQUESTED** [review]

File: packages/themes/ecl/src/ecl-ec/components/accordion.scss (hunk 1 above)
Comment (AlexanderSchmidtCE) [reply packages/themes/ecl/src/ecl-ec/components/accordion.scss]: war damit gemeint. hab mich verschrieben

**Review by sdvg — APPROVED** [review]

**Pull request merged**